internal/fs/inode/file_test.go

```diff
@@ -1004,6 +1066,59 @@ func (t *FileTest) MultipleWritesToLocalFileWhenStreamingWritesAreEnabled() {
 	err = t.in.Write(t.ctx, []byte("hello"), 0)
 	AssertEq(nil, err)
 	AssertEq(7, t.in.bwh.WriteFileInfo().TotalSize)
+	// The inode should agree about the new mtime.
```

Comment:
shall we check for mtime too? same for WriteToEmptyGCSFileWhenStreamingWritesAreEnabled